app/components/date-buttons/component.js

```diff
@@ -1,48 +1,47 @@
-import Component from "@ember/component";
 import { action } from "@ember/object";
-import classic from "ember-classic-decorator";
+import Component from "@glimmer/component";
 import moment from "moment";
 
-@classic
-export default class DateButtons extends Component {
-  init(...args) {
-    super.init(...args);
-    this.set("choices", [
-      "this week",
-      "this month",
-      "this year",
-      "last week",
-      "last month",
-      "last year",
-    ]);
-  }
+export default class DateButtonComponent extends Component {
```

Comment:
Sorry for the nitpicking :sweat_smile: 

```suggestion
export default class DateButtonsComponent extends Component {
```

Reply:
:point_up_2:

Reply:
lool, changing